.eslintrc

```diff
@@ -72,8 +72,8 @@
 		"@wordpress/i18n-translator-comments":     "off",
 		"@wordpress/valid-sprintf":                "off",
 		"@woocommerce/dependency-group":           "off",
-		"react-hooks/exhaustive-deps":             "off",
-		"react-hooks/rules-of-hooks":              "off",
+		"react-hooks/exhaustive-deps":             "error",
```

Comment:
I think it's best to keep it to `error`. `warn` can be easily overlooked, whereas with `error`, the `eslint` check would fail and it makes the developer more aware of the possible issue (which needs to be explicitly be disabled with a `eslint-disable` comment).
In my personal experience I always set this rule to `error`, rather than `warn` for this reason :)
And it also helped by making people look at the documentation 😆